docs/get-started/setup-lightdash/connect-project.mdx

```diff
@@ -745,6 +745,22 @@ If you want to change this to be something other than the default `target` defin
 
 To read more about dbt targets, [check out the dbt docs here.](https://docs.getdbt.com/reference/dbt-jinja-functions/target)
 
+#### dbt selector 
+
+You can filter out models in your dbt project that you don't want to see in Lightdash. This is useful if you have a large 
```

Comment:
```suggestion
You can filter out models in your dbt project that you don't want to connect to Lightdash. This is useful if you have a large 
```

slight change - but I think this is different to the `table selector` which is where you actually hide what you "see" in Lightdash. This is more about what you "connect" to Lightdash I think?